### ISSUE TITLE
Would like move the modal

### ISSUE THREAD

**Issue by the1220** [nyroDev/nyroModal#205]
Would like to move the modal the left and right side of the page.

Thank you for this .. this is one the best modal out there .. you rock!!

**Comment by PandaWood** [comment]
Yes, make it "draggable". I came here to find if it's already possible? 
Lightbox can do it with an extra plugin - see it in action here - http://www.picssel.com/playground/jquery/fancyboxDraggableScrollBars_25may12BIS.html

**Comment by Blackskyliner** [comment]
I really don't think a "modal" in its definition would have any need to be draggable, as it most of the time is the only you'll see to get the attention of the user you fade yout anything behind it...

@the1220 do you mean to position it not only in the entered middle of the screen? Any use-case for such a feature? Like said its a modal not a notification or anything else...

Also it somehow duplicates #145 as far as I get the request.

**Comment by nyroDev** [comment]
both issues are not exaclty the same.
#205 would like to make it draggable
#145 would like to change the position to not position it in the center.
They are both related (#205 depends on #145 I think)
